src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/CommandBarTests/UnoSamples_Tests.NativeCommandBar.cs

```diff
@@ -232,5 +232,41 @@ public void When_Navigated_CommandBarShouldHideBackButtonTitle_NativeFrame()
 			// Removing the global style we added for the CommandBar preventing other UITest to fail
 			_app.FastTap("UnsetGlobalStyleButton");
 		}
+
+		[Test]
+		[AutoRetry]
+		[ActivePlatforms(Platform.iOS)]
+		public void When_Navigated_CommandBarDisplayCustomBackButtonIcon_NativeFrame()
+		{
+			Run("UITests.Windows_UI_Xaml_Controls.CommandBar.BackButtonImage.CommandBar_Frame");
+
+			_app.WaitForElement("NavigateInitialButton");
+
+			// Will set the global style for the CommandBar to remove the Back Button Title
+			_app.FastTap("SetGlobalStyleButton");
+
+			_app.FastTap("NavigateInitialButton");
+
+			_app.WaitForElement("NavigateToPage2Button");
+			_app.FastTap("NavigateToPage2Button");
+
+			_app.WaitForElement("BackButtonImageLoaderButton");
+
+			_app.FastTap("BackButtonImageLoaderButton");
+
+			_app.Wait(TimeSpan.FromMilliseconds(500));
```

Comment:
What are you waiting for, here? Can you use `WaitForElement()` instead?

Reply:
@davidjohnoliver  The `FastTap("BackButtonImageLoaderButton");` will get the image from the `BackButton` and place it into the `Image` control that we later use for the validation. This operation takes some time and I want to make sure it has finished before doing the Screenshot